.pre-commit-config.yaml

```diff
@@ -13,3 +14,17 @@ repos:
         additional_dependencies:
           - "types-pytz"
           - "types-requests"
+        stages: [manual]
+
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        args:
+          - "--line-length=120"
```

Comment:
Any reason to override the default 88 character line length? I realize it's somewhat arbitrary what the line length is if you have an auto-formatter, but as far as I can tell most packages adhere to the default.

Reply:
Because line length doesn't correlate to any reasonable thing. It's an arbitrary rule to be followed and if you'd take a look at the reformats black did with shorter line lengths have questionable readability.